lib/models/noSignal.js

```diff
@@ -45,7 +45,8 @@ var util = require('util'),
     CHECK_INTERVAL = 8,
     context = { op: 'checkNonSignal', comp: constants.COMPONENT_NAME, corr: 'n/a', trans: 'n/a' },
     lastTime,
-    MIN_INTERVAL_MS = 30e3;
+    MIN_INTERVAL_MS = 30e3,
+    MAX_INTERVAL_MS = 2147483647;
```

Comment:
Maybe a little explanation about from where these values come as a code comment would be useful from the point of view of manteanibility.

Reply:
fixed in https://github.com/telefonicaid/perseo-fe/pull/465/commits/a5401764000550724ff9a7de83be627c7d132e27